runtime/resolvers/metricsview_time_range.go

```diff
@@ -0,0 +1,129 @@
+package resolvers
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+
+	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
+	"github.com/rilldata/rill/runtime"
+	"github.com/rilldata/rill/runtime/pkg/mapstructureutil"
+	"github.com/rilldata/rill/runtime/queries"
+)
+
+func init() {
+	runtime.RegisterResolverInitializer("metricsview_time_range", newMetricsViewTimeRangeResolver)
```

Comment:
Call it `metrics_time_range` for consistency with the other metrics-related resolvers:

```suggestion
	runtime.RegisterResolverInitializer("metrics_time_range", newMetricsViewTimeRangeResolver)
```